sdk/servicebus/service-bus/test/sessionsTests.spec.ts

```diff
@@ -602,3 +602,94 @@ describe("Peek session", function(): void {
     await peekSession(false);
   });
 });
+
+describe.skip("SessionReceiver - disconnects", function(): void {
+  afterEach(async () => {
+    return afterEachTest();
+  });
+
+  it("can receive and settle messages after a disconnect", async function(): Promise<void> {
+    const testMessage = TestMessage.getSessionSample();
+    // Create the sender and receiver.
+    sbClient = getServiceBusClient();
+    const { receiverClient, senderClient } = await getSenderReceiverClients(
+      sbClient,
+      TestClientType.UnpartitionedQueueWithSessions,
+      TestClientType.UnpartitionedQueueWithSessions
+    );
+    const receiver = receiverClient.createReceiver(ReceiveMode.peekLock, {
+      sessionId: testMessage.sessionId,
+      maxSessionAutoRenewLockDurationInSeconds: 10 // Lower this value so that test can complete in time.
+    }) as SessionReceiver;
+    const sender = senderClient.createSender();
+    // Send a message so we can be sure when the receiver is open and active.
+    await sender.send(testMessage);
+    const receivedErrors: any[] = [];
+    let settledMessageCount = 0;
+
+    let messageHandlerCount = 0;
+    let receiverIsActiveResolver: Function;
+    let receiverSecondMessageResolver: Function;
+    const receiverIsActive = new Promise((resolve) => {
+      receiverIsActiveResolver = resolve;
+    });
+    const receiverSecondMessage = new Promise((resolve) => {
+      receiverSecondMessageResolver = resolve;
+    });
+
+    // Start the receiver.
+    receiver.registerMessageHandler(
+      async (message) => {
+        console.log(`Received a message`);
+        messageHandlerCount++;
+        try {
+          await message.complete();
```

Comment:
You probably want to pass the option to registerMessageHandler to disable auto-commit here.